lib/core/src/Cardano/Wallet/Primitive/Model.hs

```diff
@@ -426,7 +429,7 @@ prefilterBlock
 prefilterBlock b u0 = runState $ do
     delegations <- mapMaybeM ourDelegation (b ^. #delegations)
     (transactions, ourU) <- foldM applyTx (mempty, u0) (b ^. #transactions)
-    return (FilteredBlock {delegations, transactions}, ourU)
+    return (FilteredBlock {delegations, poolRegistrations = b ^. #poolRegistrations , transactions}, ourU)
```

Comment:
I don't think this has its place into the **wallet** primitive model and in particular, `prefilterBlock` which role is to identify pieces that are related to our wallet (unless we allowed to register stake pool through the wallet). 

Plus, this function won't likely be "available" from the stake pool workers context because they have no wallet to filter too. Instead, I suggest to keep track of pool registrations within: https://github.com/input-output-hk/cardano-wallet/blob/master/lib/core/src/Cardano/Pool/Metrics.hs#L130

Reply:
Ah ok I forgot about the Metrics module. I have updated the description in #1090.